packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -349,8 +385,21 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
   }
 
   @override
-  void rejectGesture(int pointer) {
+  void acceptGesture(int pointer) {
+    // Ignore state 'ready' here because that would happen if this recognizer
+    // won by a sweep.
     if (pointer == primaryPointer && state == GestureRecognizerState.possible) {
+      state = GestureRecognizerState.accepted;
+    }
+  }
+
+  @override
+  void rejectGesture(int pointer) {
+    // Ignore state ready here because that would happen if this recognizer won
+    // by a sweep.
+    if (pointer == primaryPointer
+        && (state == GestureRecognizerState.possible
+            || state == GestureRecognizerState.accepted)) {
```

Comment:
nit: here too. style is to try to keep the start of expressions at the same nesting level vertically aligned. in this case i think it means the `||` goes to the end of the previous line

Reply:
Done